src/styles/app.css

```diff
@@ -0,0 +1,23 @@
+* {
+  box-sizing: border-box;
+}
+
+.App {
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 5%;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  background-color: #f4f4f4;
+}
+
+.logo {
+  height: 10%;
+  width: 20%;
+  align-self: center;
+  padding: 0%;
+  background-color: transparent;
```

Comment:
Your logo doesnt have a background, so this property isnt actually being used. You should remove it